tasks/main.yml

```diff
@@ -73,7 +73,7 @@
             ternary(__ha_cluster_cloud_agents_packages, [])
           +
           ha_cluster_fence_agent_packages }}"
-        state: present
+        state: latest
```

Comment:
If you really want to do this despite https://ansible.readthedocs.io/projects/lint/rules/package-latest/?h=package then I suggest you introduce a new variable in defaults/main.yml e.g. `ha_cluster_use_latest_packages: false` then the user can set this to `true` if they want the packages to be updated.

Reply:
Hmm this would mean we would have to split it into dnf/yum and zypper task files to accommodate differences in updates.

Reply:
@richm 
I can create new task file under both pcs and crmsh, to do package install, where var would drive latest vs present.
pcs - both yum and dnf are available, but I saw yum is not updated anymore. Let me know which one would you prefer.
- yum and dnf both have state: latest and update_only: true to do this update

crmsh - zypper module has state latest as well.

Reply:
Upgrading cluster packages by default doesn't seem to me to be a good idea. It would probably cause cluster to be restarted on all nodes at the same time, leading to loss of service.

Why would you need to split dnf / yum / zipper? Is there anything preventing using `package` module with `state: "{{ 'latest' if ha_cluster_use_latest_packages else 'present' }}"`?

Reply:
@tomjelinek Updated with if/else as you suggested and tested on SLES and RHEL.